skins/oasis/modules/templates/TemplateDraftModule_Create.php

```diff
@@ -0,0 +1,13 @@
+<section class="templatedraft-module module">
+	<h2><?= wfMessage( 'templatedraft-module-title-create' )->escaped() ?></h2>
+	<p class="templatedraft-module-subtitle"><?= wfMessage( 'templatedraft-module-subtitle-create' )->parse() ?></p>
```

Comment:
@lgarczewski Nope, `wfMessage()->parse()` parses inline, while `wfMessage()->parseAsBlock()` wraps the output in `<p></p>`.